lib/rule-context.js

```diff
@@ -72,42 +72,27 @@ var PASSTHROUGHS = [
  * @param {object} ecmaFeatures The ecmaFeatures settings passed from the config file.
  */
 function RuleContext(ruleId, eslint, severity, options, settings, ecmaFeatures) {
+    // public.
+    this.id = ruleId;
+    this.options = options;
+    this.settings = settings;
+    this.ecmaFeatures = ecmaFeatures;
+
+    // private.
+    this.eslint = eslint;
+    this.severity = severity;
```

Comment:
Oh...
I tried to freeze `context`, `context.options`, `context.settings`, and `context.ecmaFeatures` in `RuleTester`.
Then, `npm test` throws 149 errors...